src/main/java/jpass/ui/JPassFrame.java

```diff
@@ -182,6 +206,25 @@ private JPassFrame(String fileName, Locale locale) {
         this.toolsMenu.add(MenuActionType.CLEAR_CLIPBOARD.getAction());
         this.jpassMenuBar.add(this.toolsMenu);
 
+        this.settingsMenu = new JMenu(localizedMessages.getString(SETTINGS_MENU));
+        this.settingsMenu.setMnemonic(KeyEvent.VK_S);
+
+        JMenu languageMenu = new JMenu(localizedMessages.getString(SETTINGS_MENU_LANGUAGE));
+        languageMenu.setActionCommand(SETTINGS_MENU_LANGUAGE);
+        languageMenu.setIcon(getIcon("world"));
+
+        SUPPORTED_LANGUAGES.forEach((key, value) -> {
+            JMenuItem language = new JMenuItem(localizedMessages.getString(key));
+            if (Objects.equals(getCurrentLanguage(), value)) {
+                language.setIcon(getIcon("check_mark"));
```

Comment:
I've added two new icons. Let me know if these ones are ok or what should be the process to add new ones.

Reply:
For me is ok.

Reply:
Hey @edimoral, could you please use symbolic icons from https://github.com/elementary/icons ?

Also, please make sure every icon has two versions in the resources:
* one without a postfix e.g. `world.svg` with fill color `fill='#666'` (this is for the light theme)
* one with the `_dark` postfix e.g. `world_dark.svg` with fill color `fill='#999'` (this is for dark theme)